code/modules/hivemind/machines.dm

```diff
@@ -175,7 +219,8 @@
 		. = ..()
 		take_damage(I.force)
 	else
-		visible_message(SPAN_WARNING("[user] is trying to hit the [src] with [I], but it seems useless."))
+		if(prob(25))
```

Comment:
this probably shouldn't be luck based.

I recommend implementing Resistance, used on creatures, windows and airlocks. Reduce incoming damage by a flat amount